# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost.html
# Commit: 8e0aea33f622
# Full Hash: 8e0aea33f622a76c5c8baa7dcf4485ffefd1662f
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2024-07-09 09:45:28
# Regressor Bug: 1892542
# File Overlap Count: 1
# Description:
#   Bug 1892542 - Webgl e.g. createBuffer() made infallible. r=gfx-reviewers,webidl,saschanaz,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D208050
# ==============================================================================

diff -r b953e0ccb68b -r 8e0aea33f622 dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost.html
--- a/dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost.html	Mon Jul 08 19:36:12 2024 +0300
+++ b/dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost.html	Mon Jul 08 17:24:12 2024 +0000
@@ -33,11 +33,11 @@
         return;
     }
 
-    extension = gl.getExtension("WEBGL_lose_context");
+    WEBGL_lose_context = gl.getExtension("WEBGL_lose_context");
     // need an extension that exposes new API methods.
     OES_vertex_array_object = gl.getExtension("OES_vertex_array_object");
-    if (extension == null || OES_vertex_array_object == null) {
-        testFailed("Some tests failed");
+    if (WEBGL_lose_context == null || OES_vertex_array_object == null) {
+        testFailed("extension && OES_vertex_array_object failed");
         finishTest();
         return;
     }
@@ -45,7 +45,7 @@
     // We need to initialize |uniformLocation| before losing context.
     // Otherwise gl.getUniform() when context is lost will throw.
     uniformLocation = gl.getUniformLocation(program, "tex");
-    extension.loseContext();
+    WEBGL_lose_context.loseContext();
 
     canvas.addEventListener("webglcontextlost", function() {
         if (testLostContextWithoutRestore()) {
@@ -53,7 +53,7 @@
             finishTest();
             return;
         } else {
-            testFailed("Some tests failed");
+            testFailed("testLostContextWithoutRestore failed");
             finishTest();
             return;
         }